src/Modules/Workspaces/Commands.Workspaces/GetPowerBIWorkspace.cs

```diff
@@ -131,23 +140,35 @@ public override void ExecuteCmdlet()
                 this.Filter = $"tolower(name) eq '{this.Name.ToLower()}'";
             }
 
-            if (this.All.IsPresent && this.Scope == PowerBIUserScope.Organization)
-            {
-                this.ExecuteCmdletWithAll();
-                return;
-            }
-
             if (!string.IsNullOrEmpty(this.User) && this.Scope == PowerBIUserScope.Organization)
             {
                 var userFilter = $"users/any(u: tolower(u/emailAddress) eq '{this.User.ToLower()}')";
                 this.Filter = string.IsNullOrEmpty(this.Filter) ? userFilter : $"({this.Filter}) and ({userFilter})";
             }
 
+            this.GetWorkspaces();
+        }
+
+        private void GetWorkspaces()
+        {
             using (var client = this.CreateClient())
             {
+                bool defaultingFirst = false;
```

Comment:
Is it possible to move this default to 100 logic to a protected method on the base PowerBIGetCmdlet class? This seems like it should be shared for all cmdlets that implement All.